src/rp2_common/pico_platform/include/pico/platform.h

```diff
@@ -138,14 +138,14 @@ extern "C" {
  * Data marked this way will retain its value across a reset (normally uninitialized data - in the .bss
  * section) is initialized to zero during runtime initialization
```

Comment:
Ugh, I keep forgetting that GitHub doesn't show replies in "resolved" conversations.

"Looks like Luke's change here has also been reverted @kilograham ? :confused: "

Reply:
Luke's change seems to have disappeared (lost in a rebase?) but IIRC he changed it to something like:
"Data marked this way will retain its value across a reset (normally uninitialized data - in the .bss section - is initialized to zero during runtime initialization)"